server/controller/index.js

```diff
@@ -0,0 +1,59 @@
+const model = require('../model/index');
+const Alert = require('../model/alerts');
+const Preparation = require('../model/preparation');
+
+exports.getAlert = async (req, res) => {
+  try {
+    // const userInput = req.params;
+    // const alert = await Alert.findOne({city: 'Sacramento'}); 
+    const alert = await Alert.find(); 
+    res.status(200).send(alert);
+  } catch (error) {
+    console.log('error', error); //eslint-disable-line no-console
+    res.status(500);
+    res.send({error: error.errors})
+  }
+}
+
+exports.seedAlerts = (req, res) => {
+  const emergencyAlert = {
+    "city" : "Sacramento",
+    "title" : "Possibility wildfires will start and spread easily in Sacramento Valley",
+    "type" : "Wildfire",
+    "expires" : "4 days and 20 hours",
+    "description" : "Two massive fires have started in San Joaquin County,\nCalifornia. Winds are blowing north with gusts up to 50mph. The\nupdated warning indicates the possibility that fires start\nin Sacramento County. Northern California’s Red Flag Warning\nwill last into Sunday.",
+    "severity" : "warning"
+  };  
+  const newAlert = new Alert(emergencyAlert);
+  newAlert.save();
+  res.send('Database seeded!');
```

Comment:
Always remember to send a status code in response, since this is a new resource you should send back 201.